sklearn/feature_selection/_rfe.py

```diff
@@ -706,7 +726,27 @@ def fit(self, X, y, groups=None):
         self.estimator_ = clone(self.estimator)
         self.estimator_.fit(self.transform(X), y)
 
-        # Fixing a normalization error, n is equal to get_n_splits(X, y) - 1
-        # here, the scores are normalized by get_n_splits(X, y)
-        self.grid_scores_ = scores[::-1] / cv.get_n_splits(X, y, groups)
+        # reverse to stay consistent with before
+        scores_rev = scores[:, ::-1]
+        self.cv_results_ = {}
+        self.cv_results_["mean_score"] = np.mean(scores_rev, axis=0)
+        self.cv_results_["std_score"] = np.std(scores_rev, axis=0)
```

Comment:
I think that @thomasjpfan mentioned in its previous post was to change both entries using `mean_test_score` and `std_test_score` here.

Reply:
I'm sorry, I was mistaken about that.
Would it be okay to just change the names?

Reply:
yes it should be enough

Reply:
I understand. Thank you so much.